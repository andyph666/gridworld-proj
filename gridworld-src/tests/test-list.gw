int i = 0;
node foo{
	list("do it", "give up");
}
main{
	goto(foo);
}