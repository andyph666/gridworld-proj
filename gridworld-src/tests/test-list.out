
1: do it
2: give up

